_i18n/en.yml

```diff
@@ -156,15 +156,18 @@ security_page:
       us](site.baseurl/contact). You can also visit our [open-source repository](https://github.com/18F/identity-idp){:target="_blank"}.
 contact_page:
   content: |-
-    We’re experiencing extremely high traffic due to the Trusted Traveler Program launch today. Thank you for your patience as we work to improve our system performance.
+    ## Note for Trusted Traveler Program users:
+    To sign into the Trusted Traveler Program website, **you must create a new login.gov account.** login.gov does not have access to old GOES username, userID, or password and will not automatically recognize your email address. **Only your newly created login.gov username and password will allow you to sign in.**
```

Comment:
perhaps "login.gov does not have access to _your_ old GOES...."

Reply:
good note, umm not sure how adding line breaks will affect the md, but worth a try

Reply:
In my experience, the only time it gets wobbly is in LIs sometimes. Single breaks are seen as nothing more than a space.